tools/enso4igv/.gitignore

```diff
@@ -1,2 +1,4 @@
 *.vsix
 nbcode
+!.idea
```

Comment:
I believe this is needed so that `./run git-clean` does not remove the `.idea` directory here. I have been playing around with this Maven subproject, and it was more comfortable for me to open it in IDEA, rather than in VSCode.

Reply:
OK.